app/models/header_lookup.rb

```diff
@@ -1,15 +1,11 @@
 class HeaderLookup
-  LCSH_BASE_URI = "http://id.loc.gov/authorities/subjects/suggest/?q="
```

Comment:
These should have been removed before, but I think they were kept in to keep some specs working.